doc/reference/reference_lua/popen.rst

```diff
@@ -32,7 +32,9 @@ object:
 
 Below are tables with summaries of ``popen`` constructors, constants, mrthods, and fields.
 
-Popen module constructors:
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                      popen module constructors
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
Should we really place these ToC tables into sub-sections? It is rendered this way:

![image](https://github.com/user-attachments/assets/f467c9c6-85dc-4f1b-95a9-07195051de88)

Maybe we can do it in the following way (add the 'summary' words)?

* Overview
  * popen module constructors summary
  * popen module constants summary
  * popen handle methods summary
  * popen handle fields summary
* popen module constructors
* popen module constants
* popen handle methods
* popen handle fields

It seems les confusing.